fastly/block_fastly_service_v1_settings.go

```diff
@@ -86,13 +86,13 @@ func (h *SettingsServiceAttributeHandler) Register(s *schema.Resource) error {
 		Type:        schema.TypeBool,
 		Optional:    true,
 		Default:     false,
-		Description: "The default hostname",
+		Description: "Whether or not stale if error should be true",
```

Comment:
I would prefer us to use the same language as on the developer reference for consistency/accuracy:

> Enables serving a stale object if there is an error